packages/fields/src/types/Text/views/Filter.js

```diff
@@ -10,26 +10,26 @@ import { gridSize } from '@arch-ui/theme';
 
 type Props = FilterProps<string>;
 
-type State = { isInsesitive: boolean };
+type State = { isSensesitive: boolean };
 
 export default class TextFilterView extends Component<Props, State> {
   constructor(props: Props) {
     super(props);
-    this.state = { isInsesitive: this.props.field.getFilterInsensitive() };
+    this.state = { isSensesitive: this.props.field.getFilterSensitive() };
   }
 
   handleChange = ({ target: { value } }: Object) => {
     this.props.onChange(value);
   };
 
   checkboxOnChange = ({ target: { checked } }: Object) => {
-    this.setState({ isInsesitive: checked });
-    this.props.field.setFilterInsensitive(checked);
+    this.setState({ isSensesitive: checked });
+    this.props.field.setFilterSensitive(checked);
   };
 
   render() {
     const { filter, field, innerRef, value } = this.props;
-    const { isInsesitive } = this.state;
+    const { isSensesitive } = this.state;
```

Comment:
is this a typo? should this be `isSensitive`?

Reply:
is this still a checkbox? or per field setting now?

Reply:
Not a typo. Perhaps a slightly confusing double negative but it's correct.

Reply:
O Yes. It is typo. Should be isSensitive.